app/controllers/api/v1/reviews_controller.rb

```diff
@@ -14,6 +14,13 @@ def create
     end
   end
 
+  def show
+    likes = Review.find(params[:id]).tally_likes
+    dislikes = Review.find(params[:id]).tally_dislikes
```

Comment:
It would be better to set up a variable `review = Review.find(params[:id])` and then use that variable for lines 18 & 19 rather than have to query the DB twice